documentation/docs/installation.mdx

```diff
@@ -11,6 +11,18 @@ which must be installed as a dependency in the client application that uses Reve
 npm install @cognite/reveal @cognite/sdk
 ```
 
+If you will be using ThreeJS directly from the application, you will also need to install the correct version of ThreeJS and
+types information if you use TypeScript:
+
+```bash npm2yarn
+npm install three@<version>
+npm install --save-dev @types/three@<version>
+```
+
+Please find the correct version of ThreeJS in [the release documentation on Github](https://github.com/cognitedata/reveal/releases/).
+
+# Regarding WebWorkers
```

Comment:
```suggestion
## Regarding WebWorkers
```

The top-level header is described at the top of every file as

> title: Installation

It also semantically converts to h1, h2, h3 etc from (`#`, `##`, `###`). Normally a single article should have one h1 header which describes the whole article.